iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -3,12 +3,19 @@
 
 namespace Microsoft.Azure.Devices.Common.Exceptions
 {
+    /// <summary>
```

Comment:
I don't have any more context that I can add to these error codes and it seemed self-explanatory to an extent. If anyone has any insights to share, I can include them.